app/models/comment.rb

```diff
@@ -1,4 +1,11 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] Great job so far. Please like the previous reviewer recommended and clearly explained, he suggested that you specify the `author` association so it map to instances of the `User class`
instead of .... `belongs_to :user, class_name: 'User'`
you should have..... `belongs_to :author, class_name: 'User'`

please kindly correct for the comment and like model.